.github/workflows/backend.yml

```diff
@@ -479,6 +479,7 @@ jobs:
 
       # only if `backend-typing` succeeds should we try and trim the blocklist
       - run: |
+          python3 -m tools.mypy_helpers.remove_unneeded_type_ignores
```

Comment:
I didn't do this because it also introduces lint errors and slows down this suite by 50%

Reply:
> it also introduces lint errors

can you elaborate on this scenario?

Reply:
at the very least it always adds trailing whitespace, but also sometimes breaks noqa comments, etc.  it's extremely quick-and-dirty and I didn't invest time in actually making it production-worthy